ipc/cli/src/commands/wallet/list.rs

```diff
@@ -0,0 +1,42 @@
+// Copyright 2022-2024 Protocol Labs
+// SPDX-License-Identifier: MIT
+//! Wallet list cli handler
+
+use async_trait::async_trait;
+use clap::Args;
+use ipc_wallet::{EthKeyAddress, EvmKeyStore};
+use std::fmt::Debug;
+
+use crate::{get_ipc_provider, CommandLineHandler, GlobalArguments};
+
+pub(crate) struct WalletList;
+
+#[async_trait]
+impl CommandLineHandler for WalletList {
+    type Arguments = WalletListArgs;
+
+    async fn handle(global: &GlobalArguments, _: &Self::Arguments) -> anyhow::Result<()> {
+        let provider = get_ipc_provider(global)?;
+        let wallet = provider.evm_wallet()?;
```

Comment:
Its not clear to me that this command only support evm wallet types, could we let it handle fvm wallet type as well?

Reply:
Thanks. Added support for FVM now.